packages/forge2d/lib/src/dynamics/body.dart

```diff
@@ -57,7 +57,7 @@ class Body {
 
   double linearDamping = 0.0;
   double angularDamping = 0.0;
-  double gravityScale = 0.0;
+  Vector2 gravityScale = Vector2.all(0);
```

Comment:
This needs to be null by default, we can't initialize potentially unused objects on the body.